service_container/alias_private.rst

```diff
@@ -252,6 +269,26 @@ The following example shows how to inject an anonymous service into another serv
             </services>
         </container>
 
+    .. code-block:: php
+
+        // config/services.php
+        namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+        use App\Foo;
+        use App\AnonymousBar;
+
+        return function(ContainerConfigurator $configurator) {
+            $container = $configurator->services();
+
+            $container->set(Foo::class)
+                ->args([inline(AnonymousBar::class)])
+        };
+
+.. note::
+
+    Anonymous services do *NOT* inherit the definitions provided from the defaults defined in the configuration. So you'll
+    need to explicitly mark service as autowired or autoconfigured when doing an anonymous service e.g.: `inline(Foo::class)->autowire()->autoconfigure()`.
```

Comment:
We use double back ticks for inline formatting:
```diff
- `inline(Foo::class)->autowire()->autoconfigure()`
+ ``inline(Foo::class)->autowire()->autoconfigure()``
```

Reply:
kk